Dockerfile

```
@@ -1,4 +1,4 @@
-FROM google/dart:1.24.3 as build
+FROM drydock-prod.workiva.net/workiva/dart2_base_image:0.0.0-dart2.13.4 as build
```

This should just be :1 so it updates in the future
```suggestion
FROM drydock-prod.workiva.net/workiva/dart2_base_image:1
```